testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -294,6 +295,14 @@ public void testDeleteAccessControl()
         assertAccessAllowed("DELETE FROM orders", privilege("orders", SELECT_COLUMN));
     }
 
+    @Test
+    public void testUpdateAccessControl()
+    {
+        assertAccessDenied("UPDATE orders SET orderkey=123", "Cannot update columns \\[\\[orderkey]] in table *.*.*", privilege("orders", UPDATE_TABLE));
+        assertThatThrownBy(() -> getQueryRunner().execute(getSession(), ("UPDATE orders SET orderkey=123")))
+                .hasMessageContaining("This connector does not support updates");
```

Comment:
may be `.isUnsupported()` like  here https://github.com/starburstdata/starburst-presto/pull/2156/files#diff-1ef3cd4b95feec3e7fb4da15cd4bcdfb3e63952398d67d33c3b5c9caa83b7871R634